blogposts/announcing-sourcegraph-3.5.md

```diff
@@ -0,0 +1,235 @@
+
+## Powering code alerts with saved searches
+
+<p class="container">
+<div style="padding:56.25% 0 0 0;position:relative;">
+  <iframe src="https://player.vimeo.com/video/342111852?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+</div>
+<p style="text-align: center"><a href="https://vimeo.com/342111852" target="_blank">View on Vimeo</a></p>
+</p>
+
+[Saved searches](https://docs.sourcegraph.com/user/search/saved_searches) can act as bookmarks to frequently used searches, and when combined with email notifications, can help monitor critical parts of your code, organization wide.
+
+Saving a search is faster and easier, and is now accessed from the user and organization settings page.
+
+## New `orgs` field to optimize repository syncing for GitHub organizations
+
+For customers using GitHub with 1,000+ repositories, Sourcegraph would sometimes exceed GitHub’s search API rate limit during syncing, resulting in an incomplete set of cloned repositories.
+
+To address this, a [new `orgs` field](https://docs.sourcegraph.com/admin/external_service/github#selecting-repositories-for-code-search) has been added for customers not needing to filter the list of repositories for their organization. This uses a different GitHub API not subject to rate limiting.
+
+The `orgs` field is a list of GitHub organizations:
+
+```
+“orgs”: [
+    “gorilla”
+]
+```
+
+If filtering the list of repositories is required, e.g. `archived:no  forked:no` the `repositoryQuery` should be used instead of `orgs`. If the result of a query entry in `repositoryQuery` exceeds 1,000 repositories, it will need to be split out over multiple entries. Please [contact support](mailto:support@sourcegraph.com) if you require assistance.
+
+![](/blog/3.5-org-settings.png)
+
+![](/blog/3.5-org-repos.png)
+
+## Repository syncing status for site admins
+
+<p class="container">
+<div style="padding:56.25% 0 0 0;position:relative;">
+  <iframe src="https://player.vimeo.com/video/343706613?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+</div>
+<p style="text-align: center"><a href="https://vimeo.com/343706613" target="_blank">View on Vimeo</a></p>
+</p>
+
+We want customers to have greater visibility into the status of repository syncing and updating operations. in 3.5, we’re experimenting with an optional new status indicator in the navigation bar.
+
+To enable the repository syncing status indicator, add this to your site configuration:
+
+```
+"experimentalFeatures": { "statusIndicator": "enabled" }
+``` 
+
+Due to its experimental status, Sourcegraph instances with 500+ repositories may experience inaccurate status updates.
```

Comment:
Is "inaccurate" right here? I thought it was more an issue of it being "confusing" or "not useful"?

Reply:
Good catch! Updating...